app/services/idv_attempter.rb

```diff
@@ -0,0 +1,34 @@
+class IdvAttempter
+  attr_accessor :current_user, :user_session
```

Comment:
I see. In that case, I would recommend updating the tests to reflect how the class is used. Instead of setting the hash in the test, you would instantiate an instance of the class by passing in the desired arguments. For example:
```ruby
user = User.new
user_session = { idv_attempts: 3 }

attempter = IdvAttempter.new(user, user_session)

expect(attempter.exceeded?).to eq true
```
That way, you are testing the behavior, and not the implementation. The tests will work whether the class uses instance variables in the initializer or `self`.